solutions/gold/cf-1666F.mdx

```diff
@@ -117,11 +117,11 @@ void solve() {
 			// k in the even position and b[i].second - k in the odd
 			for (ll k = 0; k <= 1; k++) {
 				if (j + k > n / 2) { continue; }
-				dp[size + b[i].second][j + k] = (
-					dp[size + b[i].second][j + k] +
-					choose(positions, b[i].second - k, MOD) *
-					dp[size][j] % MOD
-				) % MOD;
+				dp[size + b[i].second][j + k] =
+				    (dp[size + b[i].second][j + k] +
+				     choose(positions, b[i].second - k, MOD) * dp[size][j] %
+				         MOD) %
+				    MOD;
```

Comment:
alright this is just straight unreasonable

Reply:
I tried to make it less but now its making the indent smaller.
Im not sure how to increase the line limit (or if I can even do this from my end).

Reply:
oh wait nvm i think it worked